src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSecurityTokenHandler.ValidateToken.Internal.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+namespace Microsoft.IdentityModel.Tokens.Saml
+{
+    /// <summary>
+    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Saml Tokens. See: http://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf
+    /// </summary>
+    public partial class SamlSecurityTokenHandler : SecurityTokenHandler
```

Comment:
do we want this to be public now?

Reply:
Partial classes need to have the same access modifier otherwise, we get a [CS0262](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/partial-declarations?f1url=%3FappId%3Droslyn%26k%3Dk(CS0262)#partial-types): Partial declarations of type have conflicting accessibility modifiers. So, as long as we keep the methods internal, we should be fine.